Editor/DAEAnimations.cs

```diff
@@ -241,7 +241,7 @@ public DaeAnimations (H3D.H3D Scene, int MdlIndex, int AnimIndex = -1)
 
                                 switch (i) {
                                     case 0:
-                                        StrTrans = DAEUtils.VectorStr (QuatTransform.GetTranslationValue (Frame));
+                                        StrTrans = DAEUtils.VectorStr (QuatTransform.GetTranslationValue (Frame) * .01f);
```

Comment:
What does this resolve?

Reply:
That transforms model scale to unity FBX units

Reply:
Should this be using the scale factor that we set in the config?

Reply:
Sorry for delay, yes!